File: DK08.py
a, op, b = input().split()

a = float(a)
b = float(b)

if op == '+':
    result = a + b
    print(f"{result:.2f}")

elif op == '-':
    result = a - b
    print(f"{result:.2f}")

elif op == '*':
    result = a * b
    print(f"{result:.2f}")

elif op == '/':
    if b == 0:
        print("Math Error")
    else:
        result = a / b
        print(f"{result:.2f}")

File: DK08/DK08.cpp
#include<bits/stdc++.h>
using namespace std;

int main() {
    string a_str, op, b_str;
    cin >> a_str >> op >> b_str;
    double a = stod(a_str);
    double b = stod(b_str);
    if (op == "+") {
        double result = a + b;
        cout << fixed << setprecision(2) << result << endl;
    } else if (op == "-") {
        double result = a - b;
        cout << fixed << setprecision(2) << result << endl;
    } else if (op == "*") {
        double result = a * b;
        cout << fixed << setprecision(2) << result << endl;
    } else if (op == "/") {
        if (b == 0) {
            cout << "Math Error" << endl;
        } else {
            double result = a / b;
            cout << fixed << setprecision(2) << result << endl;
        }
    }
    return 0;
}
